### PR TITLE
Support indian names

### LINKED ISSUE #112: Indian Name like S/O or D/O does not work
Image

### PR DESCRIPTION
**Pull request by orbitalsqwib**
Resolves #112.

This is a basic fix, as supporting quote strings will introduce a number of complex edge cases that are not a priority at the moment.

Screenshot 2025-10-21 at 16 04 23

### CODE REVIEW

File: src/test/java/casetrack/app/model/person/NameTest.java [hunk 1]
```diff
@@ -36,6 +36,7 @@ public void isValidName() {
         assertTrue(Name.isValidName("peter the 2nd")); // alphanumeric characters
         assertTrue(Name.isValidName("Capital Tan")); // with capital letters
         assertTrue(Name.isValidName("David Roger Jackson Ray Jr 2nd")); // long names
+        assertTrue(Name.isValidName("S/O Aman Sharma")); // names with slashes
```
[the remark this answers is not in the document]
Comment (orbitalsqwib) [reply src/test/java/casetrack/app/model/person/NameTest.java]: yep, sounds good - modified the test case to the proposed name

**Review by hwennnn — APPROVED** [review]
LGTM!

File: src/main/java/casetrack/app/model/person/Name.java [hunk 2]
```diff
@@ -10,13 +10,14 @@
...
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "Names should only contain alphanumeric characters, spaces, or slashes without a preceding space,"
+                    + " and it should not be blank";
 
     /*
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}//][\\p{Alnum}// ]*";
```
Comment (AppDevIn) [inline src/main/java/casetrack/app/model/person/Name.java]: I think we should change the validation string, notice it also allows /Ravi or Ravi/

Comment (orbitalsqwib) [reply src/main/java/casetrack/app/model/person/Name.java]: I don't know if we want to gate it purely to very specific cases?

[the remark this answers is not in the document]
Comment (orbitalsqwib) [reply src/main/java/casetrack/app/model/person/Name.java]: restricting this PR to support just indian names as most prior comments suggest possibility of large number of edge cases.

**Review by AppDevIn — APPROVED** [review]
Thanks for the change LGTM

**Review by lhurr — APPROVED** [review]
Left a comment, otherwise LGTM
